ec_api/apps/users/views.py

```diff
@@ -80,11 +80,24 @@ def get(self, request, *args, **kwargs):
         user = get_user(request)
         if user:
             login(request, user)
+            if not user.name:
```

Comment:
Not too important, but just because I dont like nested `if` statements I'd flip the conditionals here to return early when there isn't a user so that you avoid the nested ones:
```
if not user:
    return super().get(request, *args, **kwargs)

login(request, user)
if not user.name:
    return redirect("users:add_profile_details")
return redirect("users:profile")
```